ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/parts/base/BaseView.java

```diff
@@ -69,168 +53,22 @@ public void onBlur(BlurEvent event) {
-
-        addToolButton(maximizeButton);
-
-        if (maximizeButton.getElement() instanceof elemental.dom.Element) {
-            Tooltip.create((elemental.dom.Element) maximizeButton.getElement(),
-                    PositionController.VerticalAlign.BOTTOM, PositionController.HorizontalAlign.MIDDLE, "Maximize panel");
-        }
-    }
-
-    /**
-     * Adds part menu button.
-     */
-    private void addMenuButton() {
-        final ToolButton menuButton = new ToolButton(FontAwesome.COG + "&nbsp;" + FontAwesome.CARET_DOWN);
-        menuButton.getElement().setAttribute("name", "workBenchIconMenu");
-        menuButton.addClickHandler(event -> {
-            int left = getAbsoluteLeft(menuButton.getElement());
-            int top = getAbsoluteTop(menuButton.getElement());
-            delegate.onPartMenu(left, top + 21);
-        });
-
-        toolbarHeader.addEast(menuButton, 25);
-
-        if (menuButton.getElement() instanceof elemental.dom.Element) {
-            Tooltip.create((elemental.dom.Element) menuButton.getElement(),
-                    PositionController.VerticalAlign.BOTTOM, PositionController.HorizontalAlign.MIDDLE, "Panel options");
-        }
-    }
-
-    /**
-     * Returns absolute left position of the element.
-     *
-     * @param element
-     *          element
-     * @return
-     *          element left position
-     */
-    private native int getAbsoluteLeft(JavaScriptObject element) /*-{
-        return element.getBoundingClientRect().left;
-    }-*/;
-
-    /**
-     * Returns absolute top position of the element.
-     *
-     * @param element
-     *          element
-     * @return
-     *          element top position
-     */
-    private native int getAbsoluteTop(JavaScriptObject element) /*-{
-        return element.getBoundingClientRect().top;
-    }-*/;
-
-    /**
      * Add a button on part toolbar,
      *
      * @param button button
      */
     public final void addToolButton(@NotNull IsWidget button) {
         if (button != null) {
-            toolbarHeader.addEast(button, 18);
-        }
-    }
-
-    /**
-     * Removes button from part toolbar.
-     *
-     * @param button button
-     */
-    public void removeToolButton(@NotNull IsWidget button) {
-        if (button != null) {
-            toolbarHeader.remove(button);
-        }
-    }
-
-    /**
-     * Sets button visible on part toolbar.
-     *
-     * @param button button
-     */
-    public final void showToolButton(@NotNull IsWidget button) {
-        if (button != null) {
-            toolbarHeader.setWidgetHidden(button.asWidget(), false);
-        }
-    }
-
-    /**
-     * Hides button on part toolbar.
-     *
-     * @param button button
-     */
-    public final void hideToolButton(@NotNull IsWidget button) {
-        if (button != null) {
-            toolbarHeader.setWidgetHidden(button.asWidget(), true);
+//            toolbarHeader.addEast(button, 18);
```

Comment:
Is this something that will be changed later, what is the idea of commenting this line? Could you please explain or add clarifying comment.